app/src/pages/inside/stepPage/modals/editDefectModal/editDefectModal.jsx

```diff
@@ -207,14 +163,64 @@ export class EditDefectModal extends Component {
     return getStorageItem(REPLACE_COMMENTS_TO_ALL_AVAILABILITY_NAME);
   };
 
+  getCloseConfirmationConfig = () =>
+    this.checkIfTheDataWasChanged()
```

Comment:
The nesting is big, I think it's better to write it by using `if` construction?
```
if (!this.checkIfTheDataWasChanged()) {
  return null;
}
return {
  confirmationWarning: this.props.intl.formatMessage(
    COMMON_LOCALE_KEYS.CLOSE_MODAL_WARNING,
  ),
}
```

Reply:
Fixed